assets/src/application/ui/layout/entityActionsMenu/entityMenuItems/types.ts

```diff
@@ -0,0 +1,4 @@
+export type MenuItemProps = {
+	entityType: 'datetime' | 'ticket';
```

Comment:
This will be untranslatable. Better not to use it for output strings.

Reply:
remove this: https://github.com/eventespresso/event-espresso-core/pull/2546/commits/62b59b24f731f99dcf120e00d95aeea6ce616af0